### PR TITLE
implemented .env varibale to be load

### PR DESCRIPTION
**Pull request by stgonzales**
hey, I've implemented `.env` file load for `fiber` and `http` frameworks
please let me know if there's something that could be better done.
tks

### CODE REVIEW

File: cmd/utils/utils.go [hunk 1]
```diff
@@ -57,3 +57,14 @@ func GoFmt(appDir string) error {
...
 }
+
+// GoModTidy runs "go mod tidy" in a selected directory
+func GoModTidy(appDir string) error {
+	if err := ExecuteCmd("go",
+		[]string{"mod", "tidy"},
+		appDir); err != nil {
+		return err
+	}
+
```
Comment (basokant) [inline cmd/utils/utils.go:68]:
```suggestion
	err := ExecuteCmd("go",
		[]string{"mod", "tidy"},
		appDir);

	if err != nil {
		return err
	}
```

nit change. should probably also just run go fmt

Comment (stgonzales) [reply cmd/utils/utils.go:68]: Im following the format used in previous cmds, maybe would be good to stick with same format until have some standards (?)

File: cmd/template/main.go [hunk 2]
```diff
@@ -113,6 +113,14 @@ tmp_dir = "tmp"
...
 }
 
+func EnvFileTemplate() []byte {
+  return []byte(`#Environment Variables
+  
+# Port to run the application on
+PORT="8080"
+  `)
+}
+
```
Comment (Melkeydev) [inline cmd/template/main.go:123]: This is great! However, we have another [PR](https://github.com/Melkeydev/go-blueprint/pull/51)
That is moving go-blueprint away from this style and using tmpl + embed more.
Can we make that change here as well?
Also, we should add a .env ignore in the .gitignore generator as well in this PR!

**Review by Melkeydev — CHANGES_REQUESTED** [review]

**Comment by Melkeydev** [comment]
Also there is some conflicts that need to be resolved

File: cmd/template/main.go (hunk 2 above)
Comment (stgonzales) [reply cmd/template/main.go]: good call, I've re-based and implemented as embed, also add `.env` to gitignore

**Pull request closed without merging**